# Promoters-and-enhancers-associated-with-the-histone-H3K4me1-using-data-from-a-ChIP-Seq-experiment

Reporte.html is the document that contains all the analysis.

The Reporte_todo.zip file contains all the files used for the various analyses.

Objective: 

Processing the results of the CHIP-Seq experiment; 
  
  -Familiarization and processing of BAM, BigWig and BED files.
  
  -Application of diverse bioinformatic tools.

  -Use of the UCSC Genome Browser.

  -Examination of Various Heatmap Types.

  -Interpretation and Utilization of Position Frequency Matrices (PFMs) from the JASPAR Collection.


Using data from a CHiP-Seq experiment on histone H3K4me1 obtained from the BLUEPRINT project, we seek to obtain the promoters and enhancers associated with this histone in the cell, using the data obtained in the article “Lineage-Specific Genome Architecture Links Enhancers and Non-coding Disease Variants to Target Gene Promoters” by Javierre et al. (2016), where the promoter capture Hi-C (PCHi-C) method was used to study the spatial interaction of chromatin with promoters of specific genes, which are referred to as “baits” and distal regulatory elements, such as enhancers, which interact with these promoters, called “other ends”.

According to the article, it is expected to obtain greater enrichment of enharses in mature neutrophils. Using statistical methods, the aim is to verify that, indeed, there is a significant difference between the categorical variables “enhancer” and “promoters” with said associated histone.

Finally, the enrichment of transcription factors binding in these regions is analyzed. This is based on weight matrices obtained from JASPAR bases and using permuted sequences as a control.


Conclusion: 

We observed a greater enrichment of enhancers associated with the H3K4me1 modification compared to promoters, aligning with the expected findings from the study by Javierre et al., 2016.

Additionally, we identified binding sites for transcription factors on chromosome 22. However, further analysis is required to ascertain whether these binding sites have any functional implications in relation to the cell type.



Authors: 

Authors:

  Onna Nayyu Leyva Alcantara; Undergraduate student in Genomic sciences at Universidad Nacional Autonoma de México (UNAM)

  Rodrigo Aguilar Diaz ; Undergraduate student in Genomic sciences at Universidad Nacional Autonoma de México (UNAM)

  Aura Belem Aguilar Roldan ; Undergraduate student in Genomic sciences at Universidad Nacional Autonoma de México (UNAM)

Some files provided:

  Dr. Alejandra Medina Rivera (Principal Investigator at the International Laboratory for Human Genome Research).
